# request
string place
# geometry_msgs/Pose pose
---
# response
bool success